src/test/kotlin/nexters/admin/common/auth/JwtTokenProviderTest.kt

```diff
@@ -0,0 +1,36 @@
+package nexters.admin.common.auth
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.matchers.shouldBe
+import nexters.admin.common.exception.UnauthenticatedException
+import org.junit.jupiter.api.Test
+
+private const val VALID_SECRET_KEY = "testtesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttest"
+private const val INVALID_SECRET_KEY = "wrongtesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttest"
```

Comment:
KEY 값이 길어서 120자가 넘어가는데 중간에 엔터 넣는 것 어떻게 생각하시나요?
```suggestion
private const val VALID_SECRET_KEY =
        "testtesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttest"
private const val INVALID_SECRET_KEY =
        "wrongtesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttest"
```
<img width="1004" alt="image" src="https://user-images.githubusercontent.com/57135043/216501519-2854f22d-4521-4d03-9b77-a906e18675ef.png">

Reply:
아 그냥 최대한 길이를 줄여보겠습니다.